website/content/docs/api-gateway/consul-api-gateway-install.mdx

```diff
@@ -241,7 +244,56 @@ spec:
 
 </CodeBlockConfig>
 
-Refer to the [Kubernetes Gateway API documentation](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.Gateway) for details about configuring gateways:
+For a listener's `certificateRef` to reference a secret in a different namespace, you must also create a [ReferencePolicy](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.ReferencePolicy).
+
+The following example creates a `Gateway` named `example-gateway` in `gateway-namespace`. This `Gateway` has a `certificateRef` in `secret-namespace`.
+The reference is allowed because `reference-policy` in `secret-namespace` lets `Gateways` in `gateway-namespace` to reference `Secrets` in `secret-namespace`.
```

Comment:
```suggestion
The reference is allowed because `reference-policy` in `secret-namespace` lets `Gateways` in `gateway-namespace` reference `Secrets` in `secret-namespace`.
```